Neos.Eel/Classes/Utility.php

```diff
@@ -36,12 +36,36 @@ public static function getDefaultContextVariables(array $configuration)
                 }
                 $currentPathBase = & $currentPathBase[$pathName];
             }
-            $currentPathBase = new $objectType();
-        }
 
+            if (strpos($objectType, '::')) {
+                if (strpos($variableName, '.')) {
```

Comment:
a `$variableName` of `.foo` would pass this, since strpos returns 0 and this is falsy - is this correct?

```suggestion
                if (strpos($variableName, '.') !== false) {
```